bin/lib/database.js

```diff
@@ -0,0 +1,201 @@
+const debug = require('debug')('Morar:database');
+const AWS   = require('aws-sdk');
+AWS.config.update({region: process.env.AWS_REGION || 'us-west-2'});
```

Comment:
`us-west-2` is still our default region, we're just overriding it in this instance to reduce latency